### PR TITLE
RUM-1844: Add device and os info to telemetry events

### PR DESCRIPTION
**Pull request by jonathanmos**
Adds device and OS information to telemetry events. While we have some device and os information inside http_agent, we require this set of properties for performance metrics

### CODE REVIEW

**Review by xgouchet — APPROVED** [review]

File: schemas/telemetry/_common-schema.json [hunk 1]
```diff
@@ -100,6 +100,48 @@
         "type": "string"
       },
       "readOnly": true
+    },
+    "telemetry": {
+      "type": "object",
+      "additionalProperties": true,
```
Comment (ncreated) [inline schemas/telemetry/_common-schema.json:106]: question/ Why also adding `"additionalProperties": true`? It looks unrelated to `device` and `os` change.

Comment (jonathanmos) [reply schemas/telemetry/_common-schema.json:106]: because the error-schema and debug-schema also add properties to the telemetry object, so this is to enable a merge

Comment (ncreated) [reply schemas/telemetry/_common-schema.json:106]: Got it 👍. Checking if this is not added by accident. Makes sense 👍.

**Review by ncreated — APPROVED** [review]

**Review by RomanWuattier — APPROVED** [review]

**Review by bcaudan — APPROVED** [review]
LGTM
